src/test/wiremock-test-client/index.ts

```diff
@@ -0,0 +1,46 @@
+import { RequestMethod, WireMockAllRequestsResponse, WireMockRequestCountResponse } from "./types";
+import * as request from "supertest";
+import TestConfig from "../test-config";
+
+/**
+ * Test client for verifying and managing requests to WireMock
+ */
+export default new class SapWireMockTestClient {
+
+  /**
+   * Verifies request to WireMock
+   *
+   * @param method request method
+   * @param path request path
+   * @param count request count
+   * @returns promise of whether request is verified or not
+   */
+  verify = async (method: RequestMethod, path: string) => {
```

Comment:
```suggestion
  verify = async (method: RequestMethod, path: string): boolean => {
```

Reply:
Done with return type of Promise<number>, since in the end this returns the count of requests matching given parameters.